Jenkinsfile

```
@@ -546,6 +546,15 @@ pipeline {
             }
           }
         }
+        stage('Publish Docker image') {
+          steps {
+            withGithubNotify(context: 'Create Docker image - Linux') {
+              dir("${BASE_DIR}"){
```

There is a need to configure the required secrets to interact with `docker.elastic.co`

```suggestion
              dir("${BASE_DIR}"){
                dockerLogin(secret: env.DOCKER_SECRET, registry: env.DOCKER_REGISTRY)
```